pkg/utils/utils.go

```diff
@@ -56,7 +59,25 @@ func GetLineInFile(filepath, match string) (res string, err error) {
 			err = closeErr
 		}
 	}()
-	return GetLineInReader(file, match)
+
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		line := scanner.Text()
```

Comment:
Our other function `GetLineInReader()` checks for an error on each loop iteration, lets match that here:
```suggestion
		if scanner.Err() != nil {
			return "", fmt.Errorf("failed to read line: %w", err)
		}
		line := scanner.Text()
```